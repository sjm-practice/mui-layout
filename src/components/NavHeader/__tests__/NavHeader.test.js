/* eslint-env jest */

import React from "react";
import { render } from "@testing-library/react";

import NavHeader from "../index";

describe("<NavHeader />", () => {
  it("Should match Snapshot - collapsed", () => {
    const { asFragment } = render(<NavHeader collapsed />);

    expect(asFragment()).toMatchSnapshot();
  });

  it("Should match Snapshot - not collapsed", () => {
    const { asFragment } = render(<NavHeader />);

    expect(asFragment()).toMatchSnapshot();
  });
});
