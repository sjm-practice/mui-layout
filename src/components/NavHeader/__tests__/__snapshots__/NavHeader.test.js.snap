// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavHeader /> Should match Snapshot - collapsed 1`] = `
<DocumentFragment>
  <div
    style="padding: 8px; transition: 0.3s;"
  >
    <div
      class="MuiAvatar-root MuiAvatar-colorDefault"
      style="width: 48px; height: 48px; transition: 0.3s;"
    />
    <div
      style="padding-bottom: 16px;"
    />
    <h6
      class="MuiTypography-root MuiTypography-h6 MuiTypography-noWrap"
    >
      Sandra Adams
    </h6>
    <p
      class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-noWrap MuiTypography-gutterBottom"
    >
      sandra_88@gmail.com
    </p>
  </div>
  <hr
    class="MuiDivider-root"
  />
</DocumentFragment>
`;

exports[`<NavHeader /> Should match Snapshot - not collapsed 1`] = `
<DocumentFragment>
  <div
    style="padding: 16px; transition: 0.3s;"
  >
    <div
      class="MuiAvatar-root MuiAvatar-colorDefault"
      style="width: 60px; height: 60px; transition: 0.3s;"
    />
    <div
      style="padding-bottom: 16px;"
    />
    <h6
      class="MuiTypography-root MuiTypography-h6 MuiTypography-noWrap"
    >
      Sandra Adams
    </h6>
    <p
      class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-noWrap MuiTypography-gutterBottom"
    >
      sandra_88@gmail.com
    </p>
  </div>
  <hr
    class="MuiDivider-root"
  />
</DocumentFragment>
`;
